### ISSUE TITLE
Build fails with Go 1.23

### ISSUE THREAD

**Issue by chenrui333** [HDT3213/rdb#37]
👋 while regression build 1.0.16 release was go 1.23.0, seeing some build failure as below:

```
  # github.com/hdt3213/rdb
  link: github.com/bytedance/sonic/ast: invalid reference to encoding/json.safeSet
```

**Comment by HDT3213** [comment]
sonic does not support  go 1.23.0. It will be fixed later

**Comment by HDT3213** [comment]
fixed in [v1.0.17](https://github.com/HDT3213/rdb/releases/tag/v1.0.17)

**Comment by dtrodrigues** [comment]
Thanks! https://formulae.brew.sh/formula/rdb
